### PR TITLE
Ensure that choice is string value

### PR DESCRIPTION
**Pull request by benrudolph**
When dealing with a choice list that is numbers, choice will be a
number and the self.choices will be strings and thus it will not find
the proper choice.

http://manage.dimagi.com/default.asp?157835#877107 @czue

### CODE REVIEW

**Comment by dimagimon** [comment]
:+1: No lint errors found.

File: corehq/apps/reports_core/filters.py [hunk 1]
```diff
@@ -197,7 +197,7 @@ def __init__(self, name, required=True, label='Choice List Filter',
         self.choices = choices or []
 
     def value(self, **kwargs):
-        choice = kwargs[self.name]
+        choice = str(kwargs[self.name])
```
Comment (millerdev) [inline corehq/apps/reports_core/filters.py]: What about unicode? Not sure if the value would ever contain non-ascii characters.

Comment (benrudolph) [inline corehq/apps/reports_core/filters.py]: good point, i'll swap it out for `unicode`

Comment (czue) [inline corehq/apps/reports_core/filters.py]: looks good to me once this is addressed

**Comment by dimagimon** [comment]
:+1: No lint errors found.

File: corehq/apps/reports_core/filters.py [hunk 2]
```diff
@@ -197,7 +197,7 @@ def __init__(self, name, required=True, label='Choice List Filter',
         self.choices = choices or []
 
     def value(self, **kwargs):
-        choice = kwargs[self.name]
+        choice = unicode(kwargs[self.name])
```
Comment (czue) [inline corehq/apps/reports_core/filters.py]: @millerdev do you think this is ok or should we be checking the type first?

Comment (millerdev) [inline corehq/apps/reports_core/filters.py]: I think fine, unless you're worried that `kwargs[self.name]` may be `None` or some other value that you don't want to return the unicode version of.

Comment (millerdev) [inline corehq/apps/reports_core/filters.py]: Was waiting for tests, which have now passed, so I'm happy to merge unless you want me to hold off.

**Pull request merged**